numberGame.js

```diff
@@ -43,8 +44,6 @@ function validGuess(userInput) {
     if (userInput > 20 || userInput < 1) {
         alert('You must choose a number between 1 and 20!')
         tries++;
-        //attn: Kamran - throws an error. attempting to remove all box.textContent messages when out of guess range.
-        !box.textContent.classList.add('hidden');
```

Comment:
i really liked that you guys went for the stretch! 

ill take a look at this and let you know what i come up with!